### PR TITLE
feat: librarian role

### PR DESCRIPTION
**Pull request by Golbolco**
librarian from EM

new "quiet" effect that only allows you to whisper

might create a quiet modifier too? or give power to fungoid? TBD

### CODE REVIEW

File: Games/types/Mafia/effects/Quiet.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    message.parseForReview = this.parseForReview;
+  }
+
+  speakQuote(quote) {
+    quote.recipients = [this.player];
+    quote.modified = true;
+    quote.parseForReview = this.parseForReview;
+  }
+
+  speakWhisper(message) {
```
Comment (pikulet) [inline Games/types/Mafia/effects/Quiet.js:21]: this means he can only whisper to himself

Comment (Golbolco) [reply Games/types/Mafia/effects/Quiet.js:21]: added a check for hearing whispers

im a not sure if I should actually put the ability to hear whispers above the other checks for silencing messages, thoughts?

Comment (pikulet) [reply Games/types/Mafia/effects/Quiet.js:21]: i don't get what this means

im a not sure if I should actually put the ability to hear whispers above the other checks for silencing messages, thoughts?

what are you concerned about, and what's the intended effect

Comment (Golbolco) [reply Games/types/Mafia/effects/Quiet.js:21]: actually I think what I was asking is irrelevant, it won't make a difference

was just making sure that players with the quiet effect can still whisper even if they can't send normal messages

**Pull request closed without merging**